sklearn/feature_extraction/tests/test_text.py

```diff
@@ -562,25 +562,38 @@ def test_count_vectorizer_max_features():
 
     cv_1 = CountVectorizer(max_features=1)
     cv_3 = CountVectorizer(max_features=3)
+    cv_all = CountVectorizer(max_features=1.0)
+    cv_half = CountVectorizer(max_features=0.5)
     cv_None = CountVectorizer(max_features=None)
 
     counts_1 = cv_1.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
     counts_3 = cv_3.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
+    counts_all = cv_all.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
+    counts_half = cv_half.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
     counts_None = cv_None.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
 
     features_1 = cv_1.get_feature_names()
     features_3 = cv_3.get_feature_names()
+    features_all = cv_all.get_feature_names()
```

Comment:
I'm not sure if I understand what you mean. I already check if the new percentage option is consistent with manually specifying the amount of features (e.g. 3 = 0.5 and 1 = None) in the assert statements a few line under this. Which check do you want me to add?